pse/p0tionConfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "title": "EthStorage Trusted Setup Ceremony",
   "description": "This trusted setup ceremony is for the EthStorage circuits, which enable proof of storage for dynamic L2 datasets through a verifier contract on Ethereum L1.",
-  "startDate": "2025-07-18T00:00:00",
-  "endDate": "2025-07-23T00:00:00",
+  "startDate": "2025-08-08T00:00:00",
```

Comment:
Could you clarify the time zone used for the startDate and endDate fields?

Reply:
The default time zone is UTC.